vendor/assets/stylesheets/dvl/core/sidebar.scss

```diff
@@ -56,6 +56,11 @@
   font-size: 0.9rem;
 }
 
+.sidebar_data_value,
```

Comment:
Selector `sidebar_data_value` should be written in lowercase with hyphens<br>Selector `sidebar_data_details` should be written in lowercase with hyphens